apps/windows/wdio.conf.js

```diff
@@ -29,10 +32,10 @@ exports.config = {
   logLevel: 'info', // Level of logging verbosity: trace | debug | info | warn | error | silent
 
   // If you only want to run your tests until a specific amount of tests have failed use bail (default is 0 - don't bail, run all tests).
-  bail: 0,
+  bail: 1,
   waitforTimeout: defaultWaitForTimeout, // Default timeout for all waitForXXX commands.
   connectionRetryTimeout: defaultConnectionRetryTimeout, // Timeout for any WebDriver request to a driver or grid.
-  connectionRetryCount: 2, // Maximum count of request retries to the Selenium server.
+  connectionRetryCount: 1, // Maximum count of request retries to the Selenium server.
```

Comment:
Why did we bump this down?

Reply:
To match Win32. It was the default value so I decided to leave it. This was for testing purposes